models/rocifi/polygon/rocifi_polygon_v2_loan_master.sql

```diff
@@ -0,0 +1,302 @@
+{{config (
+    alias = 'v2_loan_master',
+    post_hook = '{{ 
+        expose_spells(\'["polygon"]\',
+        "project", 
+        "rocifi",
+        \'["maybeyonas"]\') }}'
+)}} with score as (
+    select evt_block_time,
+        score,
+        tokenId as token_id
+    from {{source('rocifi_v2_polygon', 'ScoreDB_evt_ScoreUpdated')}}
+),
+NFCS as (
+    select m.evt_block_time as mint_time,
+        _recipient as minter,
+        _tokenId::int as token_id,
```

Comment:
We can use cast() instead of `::`
```suggestion
        cast(_tokenId as int) as token_id,
```